Sources/Workspace/UserToolchain.swift

```diff
@@ -205,10 +205,17 @@ public final class UserToolchain: Toolchain {
         self.xctest = nil
       #endif
 
-        self.extraSwiftCFlags = (destination.target.isDarwin()
-                                    ? ["-sdk", destination.sdk.pathString]
-                                    : [])
-                                  + destination.extraSwiftCFlags
+        if destination.target.isDarwin() {
+            self.extraSwiftCFlags = ["-sdk", destination.sdk.pathString]
+        } else if (destination.target.isWindows()) {
+            self.extraSwiftCFlags = [
+              "-sdk", destination.sdk.pathString,
+              "-resource-dir", destination.sdk.pathString + "\\usr\\lib\\swift",
+              "-L", destination.sdk.pathString + "\\usr\\lib\\swift\\windows"
```

Comment:
This is the wrong place for this change. I think the Swift driver should do it.

Reply:
The swift driver cannot do this yet.  This is something which requires a large refactoring to enable that.  That work is something which is dependent on @devincoughlin upstreaming the driver changes for catalyst.  @brentdax can give you better context for that.  In the mean time, this is the best approach.  We do the same thing in the builds on Azure as well - you will find that we compute the paths and pass them through via CMake caches.

This is related to the SDK unification work that @jrose-apple, yourself, and I discussed a while ago and is still waiting on the Apple to get the changes merged before we can make progress on that.

Reply:
I see. This seems fine then. Can we put a FIXME and reference the SR for the driver tasks?

Reply:
Yes, I think it makes sense to add a comment saying that this is temporary.